jbpm-wb-human-tasks/jbpm-wb-human-tasks-client/src/main/java/org/jbpm/workbench/ht/client/editors/taskslist/AbstractTaskListPresenter.java

```diff
@@ -130,6 +135,19 @@ protected DataSetReadyCallback getDataSetReadyCallback(final Integer startRange,
                 .withEmptyResultsCallback(() -> setEmptyResults());
     }
 
+    private Set<Long> findDuplicateId(List<TaskSummary> taskSummaries) {
+        Set<Long> id = new HashSet<>();
+        return taskSummaries.stream().filter(ts -> !id.add(ts.getId())).map(TaskSummary::getId).collect(Collectors.toSet());
```

Comment:
```suggestion
        return taskSummaries.stream().map(TaskSummary::getId).collect(Collectors.toSet());
```

Reply:
This code is to find duplicate ids, not to remove duplicates